envs/example/defaults-2.0.yml

```diff
@@ -197,6 +197,28 @@ ceph:
 lvm:
   enabled: false
 
+v7k:
```

Comment:
we should default false in role defaults.. even if is in defaults-2.0.yml. Reasoning is that in production if for some reason envs is missing this role default will still default to false.